projects/ccf-api/src/lib/routes/v1/utils/parse-filter.ts

```diff
@@ -95,11 +103,19 @@ function processParameter(result: Filter, key: string, value: unknown): void {
       setIfDefined(result, 'ageRange', parseRange(value, 0, 100));
       break;
 
+    case 'age':
+      setIfDefined(result, 'ageRange', parseMinMaxRange(value, 0, 100));
+      break;
+
     case 'bmirange':
     case 'bmi-range':
       setIfDefined(result, 'bmiRange', parseRange(value, 0, 100));
```

Comment:
I'm fine with value 0, but some have higher max range. Please make sure this aligns with the ranges for bmi and age that is in the portal. (Should also show up in the open api spec I think)